frontend/src/components/CopyableField.tsx

```diff
@@ -10,17 +10,24 @@ export default function CopyableField({
   abbreviate = true,
   link,
   copy = true,
+  copyText,
   showFullValueTip = false,
 }: {
   text: string
   abbreviate?: boolean
   link: string
   copy?: boolean
+  copyText?: string
   showFullValueTip?: boolean
 }) {
+
+  if (!copyText) {
+    copyText = text
+  }
+
   function copyToClipboard(e: React.MouseEvent<HTMLButtonElement>) {
     e.stopPropagation()
-    navigator.clipboard.writeText(text)
+    navigator.clipboard.writeText(copyText)
```

Comment:
You can avoid the `if` statement here by doing:
```
navigator.clipboard.writeText(copyText ?? text)
```

`??` evaluates to the second variable if the first one is null or undefined

Reply:
Nice! Thanks for the tip.